src/main/java/training/subsystems/ArmSubsystems/elbow/neoElbow/NeoElbow.java

```diff
@@ -0,0 +1,81 @@
+package training.subsystems.ArmSubsystems.elbow.neoElbow;
+
+import com.revrobotics.CANSparkBase;
+import com.revrobotics.CANSparkLowLevel;
+import com.revrobotics.CANSparkMax;
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.geometry.Rotation2d;
+import training.GlobalConstants;
+import training.subsystems.ArmSubsystems.elbow.ElbowConstants;
+import training.subsystems.ArmSubsystems.elbow.ElbowInputsAutoLogged;
+import training.subsystems.ArmSubsystems.elbow.IElbow;
+
+
+public class NeoElbow implements IElbow {
+
+	private final CANSparkMax motor;
+
+	public NeoElbow() {
+		this.motor = new CANSparkMax(NeoElbowConstants.ID, CANSparkLowLevel.MotorType.kBrushless);
+		motorConfig();
+	}
+
+	private void motorConfig() {
```

Comment:
```suggestion
	private void configMotor() {
```
needs to be פעולה and not שם עצם